modin/engines/ray/utils.py

```diff
@@ -164,30 +133,19 @@ def initialize_ray(
 """,
             )
             object_store_memory = Memory.get()
-            plasma_directory = RayPlasmaDir.get()
             if IsOutOfCore.get():
-                if plasma_directory is None:
-                    from tempfile import gettempdir
-
-                    plasma_directory = gettempdir()
                 # We may have already set the memory from the environment variable, we don't
                 # want to overwrite that value if we have.
                 if object_store_memory is None:
                     # Round down to the nearest Gigabyte.
-                    try:
-                        system_memory = ray._private.utils.get_system_memory()
-                    except AttributeError:  # Compatibility with Ray <= 1.2
-                        system_memory = ray.utils.get_system_memory()
+                    system_memory = ray._private.utils.get_system_memory()
```

Comment:
isn't there a non-private API to use? this looks unsafe

Reply:
This is carried over from how we used it before. Should I update as a part of this PR?

Reply:
I think that if there's stable API available in Ray to get this (not a private one), and it's small enough, then it should be acceptable to call it a part of "migration to ray 1.4" 😉

Reply:
There is not such a stable API that I can find.

Reply:
so be it for now then...